Compilation instructions incomplete. "no nasm found"

Hi i tried to setup the emulator but it get stuck at `make`:

```
appending configuration tag "CXX" to libtool
appending configuration tag "F77" to libtool
configure: error: no nasm (Netwide Assembler) found
Makefile:87: recipe for target 'build/x86_64-linux-gnu/libs/libjpeg.so.8' failed
make[1]: *** [build/x86_64-linux-gnu/libs/libjpeg.so.8] Error 1
make[1]: Leaving directory '/home/moritz/kobo/koreader-dev/koreader/base'
Makefile:79: recipe for target 'base/build/x86_64-linux-gnu/luajit' failed
make: *** [base/build/x86_64-linux-gnu/luajit] Error 2
```

So i installed **nasm** (Debian Jessie) and it works. Can someone add that to the instructions? I am not sure where to add it, because it is only for x86!?

--- a/README.md
+++ b/README.md
@@ -66,10 +66,12 @@ OS. Windows users are suggested to develop in a [Linux VM][linux-vm] or use Wubi
 
 To get and compile the source you must have `patch`, `wget`, `unzip`, `git`, `autoconf`,
 `subversion` and `cmake` installed. Version of autoconf need to be greater than 2.64.
+You also need nasm and of course gcc or clang - if you want to cross-compile for other
+architecture, you need a proper cross-compile toolchain.
 
 Ubuntu users may need to run:
 ```
-sudo apt-get install build-essential libtool gcc-multilib
+sudo apt-get install build-essential libtool gcc-multilib nasm
 ```
 
 Cross compile toolchains are available for Ubuntu users through these commands:

